pkg/ebpf/tracer_windows.go

```diff
@@ -15,27 +15,42 @@ import (
 	"encoding/binary"
 	"expvar"
 	"fmt"
+	"net"
+	"sync/atomic"
 	"time"
 	"unsafe"
 
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 
+	"golang.org/x/net/ipv4"
 	"golang.org/x/sys/windows"
 )
 
 const (
 	driverFile = `\\.\ddfilter`
+
+	// Number of buffers to use with the IOCompletion port to communicate with the driver
+	totalReadBuffers = 32
```

Comment:
Will you add some documentation around when/if this value should change?

Reply:
Might we want to make this configurable so we can test different values without rebuilding the system probe ?

Reply:
I think so, but if it's okay, I'm going to add this functionality in a separate PR just to keep things simple and contained.